Sources/CryptorRSA/CryptorRSAKey.swift

```diff
@@ -318,28 +318,41 @@ extension CryptorRSA {
 				
 				throw Error(code: ERR_CREATE_CERT_FAILED, reason: "Unable to create certificate from certificate data.")
 			}
+			
+			var key: SecKey? = nil
+			#if swift(>=4.2)
+			if #available(macOS 10.14, iOS 12.0, watchOS 5.0, *) {
+				
+				key = SecCertificateCopyKey(certData)
+				
+			} 
+			#endif
+			if key == nil {
```

Comment:
Without the _#if swift(>=4.2)_ this check isn't needed and can be replace by a simple _else_ branch off the _if #available_ statement.  This'll make the code, IMHO, easier to read...

Reply:
This is required because of the `#if swift(>=4.2)`. I felt the `if key == nil ` was simpler than duplicating the code to have two if else blocks.